docs/en_US/TrialExample/KDExample.rst

```diff
@@ -19,24 +19,21 @@ PyTorch code
 
 .. code-block:: python
 
-   from knowledge_distill.knowledge_distill import KnowledgeDistill
-   kd = KnowledgeDistill(kd_teacher_model, kd_T=5)
-   alpha = 1
-   beta = 0.8
    for batch_idx, (data, target) in enumerate(train_loader):
-       data, target = data.to(device), target.to(device)
-       optimizer.zero_grad()
-       output = model(data)
-       loss = F.cross_entropy(output, target)
-       # you only to add the following line to fine-tune with knowledge distillation
-       loss = alpha * loss + beta * kd.loss(data=data, student_out=output)
-       loss.backward()
+      data, target = data.to(device), target.to(device)
+      optimizer.zero_grad()
+      y_s = model_s(data)
+      y_t = model_t(data)
+      loss_cri = F.cross_entropy(y_s, target)
 
-User configuration for KnowledgeDistill
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+      # kd loss
+      p_s = F.log_softmax(y_s/kd_T, dim=1)
+      p_t = F.softmax(y_t/kd_T, dim=1)
+      loss_kd = F.kl_div(p_s, p_t, size_average=False) * (self.T**2) / y_s.shape[0]
 
+      # total loss
+      loss = loss_cir + loss_kd
+      loss.backward()
 
```

Comment:
Indentation is three spaces, change to four?

Reply:
Thank you, changed.